common/utils/EventLogRiemannClient.js

```diff
@@ -104,6 +105,11 @@ class EventLogRiemannClient {
       if (appGuid !== undefined) {
         eventName = `${eventName}.app_guid.${appGuid}`;
       }
+
```

Comment:
unnecessary newline. See it follows the same pattern as above. So remove them even from above.

Reply:
ok , will do.